test/cpp_extensions/cudnn_extension.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include <ATen/cudnn/Descriptors.h> // for TensorDescriptor
 #include <ATen/cuda/Exceptions.h> // for CUDNN_CHECK
-#include <ATen/cudnn/Handles.h> // for getCudnnHandle
```

Comment:
The name changes for the files we already have are unnecessary.

Reply:
please discuss w/ @ezyang who requested this